lib/src/rendering/overflow_view.dart

```diff
@@ -58,19 +63,17 @@ class RenderOverflowView extends RenderBox
   }
 
   OverflowViewLayoutBehavior get layoutBehavior => _layoutBehavior;
-  OverflowViewLayoutBehavior _layoutBehavior;
+
   set layoutBehavior(OverflowViewLayoutBehavior value) {
     if (_layoutBehavior != value) {
       _layoutBehavior = value;
       markNeedsLayout();
     }
   }
 
-  bool _isHorizontal;
-
   @override
   void setupParentData(RenderBox child) {
-    if (child.parentData is! OverflowViewParentData)
+    if (child.parentData is !OverflowViewParentData)
```

Comment:
I'm not sure about this, I though the "is not a" operator was `is!` and not `is !`

Reply:
good point I tested the two version, same result, but the documentation uses `is!` you right :)